### PR TITLE
HW01 Tina Chen

### PR DESCRIPTION
**Pull request by pyctina**
I have edited README.md following the steps on the Mastering Markdown website. One of the tasks that gave me a little trouble is to embed an image, but I eventually figured that I did not copy the image link right. 

When I submitted the assignment via pull request, I followed the instructions on the homework workflow. I started by saving my changes and committing them. Then, I pushed the changes up to GitHub and created a new pull request.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,27 @@
...
+  * yoga
+  * swimming 
+* embroidery
+* binge watching
+  * some of my favorite tv shows are: 
+    1. Criminal minds
+    1. Pysch
+    1. Selling Sunset
+    1. 90 Days Fiance (*my guilty pleasure*)
+
```
Comment (April3112) [inline README.md:19]: I really love the three-layer bullet points!

**Comment by yunshu3112** [comment]
Here is the peer review from Yunshu Hu:

I really appreciate how Tina orgainized her self-introduction. She began with her basic information and then described her hobbies using bullet points. She included her Github page, which is smart!
She reflected how she completed the homework, and shared her experience with Markdown.
She included images, links, and emphasis in her file, showing her capability of using Markdown syntax.

I think she might improve in the following three aspects:
- Make more use of headers. Currently she only used headers twice, I think she can use headers with more diversity.
- Explore more Markdown syntax in the future.
- The three-layer bullet points seem not so necessary to me. Maybe two layers are enough.

Overall, I think Tina did a great job in this homework!

File: README.md [hunk 2]
```diff
@@ -1 +1,27 @@
-# hw01
\ No newline at end of file
+# hw01
+by Tina Chen
+
+### Biography
+Hi! my name is *Peiying Chen*, but I go by **Tina**. I am a first year in the college double majoring in **economics and psychology**. I came from China, and I moved to Irvine, California when I was in middle school.
```
Comment (kathleencannell) [inline README.md:5]: Good use of headers, bold, and italics!

File: README.md [hunk 3]
```diff
@@ -1 +1,27 @@
...
+Hi! my name is *Peiying Chen*, but I go by **Tina**. I am a first year in the college double majoring in **economics and psychology**. I came from China, and I moved to Irvine, California when I was in middle school. 
+
+During my free time, I enjoy doing: 
+
+* exercising 
+  * yoga
+  * swimming 
+* embroidery
+* binge watching
+  * some of my favorite tv shows are:
```
Comment (kathleencannell) [inline README.md:14]: Great use of lists, I didn't make a list on my own bio but I wish I would have, it looks really nice when viewing the file.

File: README.md [hunk 4]
```diff
@@ -1 +1,27 @@
...
+  * some of my favorite tv shows are: 
+    1. Criminal minds
+    1. Pysch
+    1. Selling Sunset
+    1. 90 Days Fiance (*my guilty pleasure*)
+
+my github website is: 
+[Tina@GitHub](https://github.com/pyctina)
+    
+![thank you](https://assets.entrepreneur.com/article/h1/thank-you.jpg)
```
Comment (kathleencannell) [inline README.md:23]: This image is a nice touch!

File: README.md [hunk 5]
```diff
@@ -1 +1,27 @@
...
+
+### Biography
+Hi! my name is *Peiying Chen*, but I go by **Tina**. I am a first year in the college double majoring in **economics and psychology**. I came from China, and I moved to Irvine, California when I was in middle school. 
+
+During my free time, I enjoy doing: 
+
+* exercising 
+  * yoga
+  * swimming 
+* embroidery
```
Comment (kathleencannell) [inline README.md:12]: I would suggest putting a picture of your embroidery art in if you wanted to, that could personalize the bio even more!

File: README.md [hunk 6]
```diff
@@ -1 +1,27 @@
...
+* binge watching
+  * some of my favorite tv shows are: 
+    1. Criminal minds
+    1. Pysch
+    1. Selling Sunset
+    1. 90 Days Fiance (*my guilty pleasure*)
```
Comment (kathleencannell) [inline README.md:18]: A non-computing suggestion is to not feel guilty about things you find pleasure in haha! But nice use of italics also:) Perhaps linking these TV shows would have been a nice touch if someone would like to know where to watch them.

Comment (kathleencannell) [reply README.md:18]: Haha perhaps you could use the strikethrough over the words "my guilty pleasure" using ~~ on either side of the words, that could have added more to the humor:)

File: README.md (hunk 2 above)
Comment (nearridge) [inline README.md:5]: Thanks for the intro!

File: README.md (hunk 5 above)
Comment (nearridge) [inline README.md:12]: Great job with this unordered list

File: README.md (hunk 4 above)
Comment (nearridge) [inline README.md:23]: Great job linking and embedding this image.

**Review by nearridge — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Improvement |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |       x    |              |            |
| **Coding strategy**         |     x      |              |            |
| **Presentation**            |           |     x         |            |
| **Achievement, creativity** |           |       x       |            |
| **Reproducibility**         |        x   |              |            |

Remarks: No major issues! Would have loved to have seen maybe some image uploads of things you do in your free time so I can see you demonstrate that skill.

**Pull request closed without merging**